### DISCUSSION TITLE
i successfully installed the program by following the instructions but it wont run

### DISCUSSION THREAD

**Discussion by merrychrishna** [acheong08/ChatGPT#486] [Q&A]
i successfully installed the program by following the instructions but it wont run. i even tried to import the module but it cant find it.

$ python3 -m revApiGPT
/usr/bin/python3: No module named revApiGPT

**Comment by wzqstudio** [comment]
your revchatgpt may be installed in your python3 path, not in the sitepackage path, add your revchatgpt path to PATH

**Reply by aiailun520** [reply]
我也有这种情况很苦恼
